build.rs

```diff
@@ -16,27 +12,63 @@ use walkdir::WalkDir;
 /// A temporary directory for proto building
 const TMP_PATH: &str = "/tmp/steward/";
 /// the output directory
-const OUT_PATH: &str = "../steward_proto_rust/src/prost/";
-
-// All paths must end with a / and either be absolute or include a ./ to reference the current
-// working directory.
+const OUT_PATH: &str = "src/gen/proto/";
 
 fn main() {
-    let out_dir = Path::new(&OUT_PATH);
-    let tmp_dir = Path::new(&TMP_PATH);
+    generate_contract_abis();
+    generate_rust_protos();
+}
 
-    println!(
-        "Compiling .proto files to Rust into '{}'...",
-        out_dir.display()
-    );
+fn generate_contract_abis() {
+    // (JSON/Contract name, output file name)
+    let contracts = vec![
+        ("AaveV2StablecoinCellar", "aave_v2_stablecoin"),
+        ("CellarV1", "cellar_v1"),
+        ("CellarV2", "cellar_v2"),
```

Comment:
Just double checking, at this point we don't have Cellar 2.2 yet, that's coming when we merge the latest release into main, yes? Will also be lots more adaptors to add.

Reply:
Yes, those will be included in the catch-up work